command/agent/consul/service_client.go

```diff
@@ -1161,7 +1149,28 @@ func (c *ServiceClient) sync(reason syncReason) error {
 	// Only log if something was actually synced
 	if sreg > 0 || sdereg > 0 || creg > 0 || cdereg > 0 {
```

Comment:
Should `fails > 0` be present here as well, or are we just letting the multierror handle that case?

Reply:
Good catch. Will fix.